File: src/Controllers/GraafController.java
package Controllers;

import java.util.ArrayList;
import java.util.List;

import Factorys.NodeFactory;
import Models.BaseNode;

public class GraafController {
	
	private List<BaseNode> nodes;
	
	public GraafController() {
		nodes = new ArrayList<BaseNode>();
		NodeFactory nodee = new NodeFactory();
		GraafParser parser = new GraafParser("C:\\tmp\\graaf.txt");
		generateNodes(parser.getInput());
	}

	public void generateNodes(List<String> input) {	
		for (String s : input) {
			if (!s.contains("#") && !s.equals("")) {
				System.out.println(s);
				nodes.add(NodeFactory.getNode(s));
			}
		}
		linkNodes();
	}
	
	public void linkNodes(){
		for(BaseNode node: nodes){
			
		}
	}
}

File: GraafMachine/GraafMachine/Controllers/GraafController.cs
using GraafMachine.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GraafMachine.Controllers
{
    class GraafController
    {
        public GraafController()
        {
            // Make parser
            GraafParser parser = new GraafParser("c:\\tmp\\graaf.txt");
            generateNodes(parser.getLines());

        }

        private void generateNodes(List<String> lines)
        {
            // Loop through all lines
            foreach(String line in lines) {
                try
                {
                    BaseNode.create("and").work();
                }
                catch (Exception e)
                {
                    Console.WriteLine("error generating");
                }
            }
        }

    }
}
